app-frontend/src/app/core/services/map.service.js

```diff
@@ -288,12 +294,21 @@ class MapWrapper {
             let boundsGeoJson = L.geoJSON();
             boundsGeoJson.addData(scene.tileFootprint);
             let imageBounds = boundsGeoJson.getBounds();
-            let overlay = L.imageOverlay(thumbUrl, imageBounds, {
-                opacity: 1,
-                attribution: `©${scene.datasource}` +
-                    ' | Previews are not representative of actual scene quality.'
-            });
-            this.setLayer('thumbnail', overlay);
+            if (!persist) {
+                let overlay = L.imageOverlay(thumbUrl, imageBounds, {
```

Comment:
Minor, but it seems like this overlay could be defined before the conditional to cut out some duplicate stuff. Up to you if you want to adjust.